packages/@lightningjs/ui-components/src/components/Artwork/Artwork.stories.js

```diff
@@ -108,36 +108,36 @@ Artwork.argTypes = {
   }
 };
 
-Artwork.parameters = {
-  argActions: {
-    foregroundSrc: (foregroundSrc, component) => {
-      component.tag('Artwork').foregroundSrc =
-        'none' !== foregroundSrc ? foregroundSrc : undefined;
-    },
-    srcCallback: (active, component) => {
-      if (active) {
-        // Accepts a regular function or function that returns a promise
-        component.tag('Artwork').patch({
-          src: '8501866671289235112',
-          srcCallback: ({ w, closestAspectRatio, src }) => {
-            return new Promise(resolve => {
-              setTimeout(() => {
-                resolve(
-                  `https://myriad.merlin.comcast.com/select/image?entityId=${src}&width=${w}&ratio=${closestAspectRatio}&rule=noTitle`
-                );
-              }, 500);
-            });
-          }
-        });
-      } else {
-        component.tag('Artwork').patch({
-          src: 'https://myriad.merlin.comcast.com/select/image?entityId=8501866671289235112&width=400&ratio=3x4&rule=noTitle',
-          srcCallback: undefined
-        });
-      }
-    }
-  }
-};
+// Artwork.parameters = {
+//   argActions: {
+//     foregroundSrc: (foregroundSrc, component) => {
+//       component.tag('Artwork').foregroundSrc =
+//         'none' !== foregroundSrc ? foregroundSrc : undefined;
+//     },
+//     srcCallback: (active, component) => {
+//       if (active) {
+//         // Accepts a regular function or function that returns a promise
+//         component.tag('Artwork').patch({
+//           src: '8501866671289235112',
+//           srcCallback: ({ w, closestAspectRatio, src }) => {
+//             return new Promise(resolve => {
+//               setTimeout(() => {
+//                 resolve(
+//                   `https://myriad.merlin.comcast.com/select/image?entityId=${src}&width=${w}&ratio=${closestAspectRatio}&rule=noTitle`
+//                 );
+//               }, 500);
+//             });
+//           }
+//         });
+//       } else {
+//         component.tag('Artwork').patch({
+//           src: 'https://myriad.merlin.comcast.com/select/image?entityId=8501866671289235112&width=400&ratio=3x4&rule=noTitle',
+//           srcCallback: undefined
+//         });
+//       }
+//     }
+//   }
+// };
```

Comment:
Can we remove this instead? It looks like this was what it was originally for IS (since it's pointing to Myriad) and was a reference for what is below (which seems to be the same thing, just different `src`s.

Reply:
Yep. I just didn't want to delete it until someone confirmed that we didn't need both for some reason.